spec/views/trips/show.html.erb_spec.rb

```diff
@@ -26,9 +27,8 @@
     @trip.update({status: 'applied'})
     render
     expect(@trip.status).to eq('applied')
-    expect(rendered).to have_content(t('status.applied'))
+    expect(rendered).to have_content('Applied')
```

Comment:
Nein, da in die trips_show angepasst ist und dort dann 'Applied' steht. Viewtests eben...